src/shared/components/templates-grid/TemplatesGrid.scss

```diff
@@ -10,5 +10,51 @@
   .spaceholder {
     width: 320px;
     height: 444px;
+    padding: 25px;
```

Comment:
I think adding classes for this is not needed - it can be just divs with simple :nth-of-type property. 
Problem with current solution is - when we change template design we need to change implementation for this. Please use simple divs without classNames